invenio/legacy/inveniocfg.py

```diff
@@ -713,11 +713,11 @@ def cli_cmd_create_demo_site(conf):
             print("ERROR: failed execution of", cmd)
             sys.exit(1)
     cli_cmd_reset_fieldnames(conf) # needed for I18N demo ranking method names
-    for cmd in ["%s/bin/webaccessadmin -u admin -c -r -D" % CFG_PREFIX,
-                "%s/bin/webcoll -u admin" % CFG_PREFIX,
-                "%s/bin/webcoll 1" % CFG_PREFIX,
-                "%s/bin/bibsort -u admin --load-config" % CFG_PREFIX,
-                "%s/bin/bibsort 2" % CFG_PREFIX, ]:
+    for cmd in ("webaccessadmin -u admin -c -a -D",
```

Comment:
If I am not wrong this part is the same than https://github.com/inveniosoftware/invenio/pull/2818/files#diff-2d38438784b9f730ac13a86cf37fa97aR59

Do you think it is possible to factorize ?